miden-lib/src/tests/test_prologue.rs

```diff
@@ -93,102 +93,108 @@ fn test_transaction_prologue() {
 }
 
 fn global_input_memory_assertions(process: &Process<MockHost>, inputs: &PreparedTransaction) {
-    // The block hash should be stored at the BLK_HASH_PTR
     assert_eq!(
         read_root_mem_value(process, BLK_HASH_PTR),
-        inputs.block_header().hash().as_elements()
+        inputs.block_header().hash().as_elements(),
+        "The block hash should be stored at the BLK_HASH_PTR"
     );
 
-    // The account ID should be stored at the ACCT_ID_PTR
-    assert_eq!(read_root_mem_value(process, ACCT_ID_PTR)[0], inputs.account().id().into());
+    assert_eq!(
+        read_root_mem_value(process, ACCT_ID_PTR)[0],
+        inputs.account().id().into(),
+        "The account ID should be stored at the ACCT_ID_PTR"
+    );
 
-    // The account commitment should be stored at the ACCT_HASH_PTR
     assert_eq!(
         read_root_mem_value(process, INIT_ACCT_HASH_PTR),
-        inputs.account().hash().as_elements()
+        inputs.account().hash().as_elements(),
+        "The account commitment should be stored at the ACCT_HASH_PTR"
     );
 
-    // The nullifier commitment should be stored at the NULLIFIER_COM_PTR
     assert_eq!(
-        read_root_mem_value(process, NULLIFIER_COM_PTR),
-        inputs.input_notes().commitment().as_elements()
+        read_root_mem_value(process, NULLIFIER_COMMITMENT_PTR),
+        inputs.input_notes().nullifier_commitment().as_elements(),
+        "The nullifier commitment should be stored at the NULLIFIER_COMMITMENT_PTR"
     );
 
-    // The initial nonce should be stored at the INIT_NONCE_PTR
-    assert_eq!(read_root_mem_value(process, INIT_NONCE_PTR)[0], inputs.account().nonce());
+    assert_eq!(
+        read_root_mem_value(process, INIT_NONCE_PTR)[0],
+        inputs.account().nonce(),
+        "The initial nonce should be stored at the INIT_NONCE_PTR"
+    );
 
-    // The transaction script root should be stored at the TX_SCRIPT_ROOT_PTR
     assert_eq!(
         read_root_mem_value(process, TX_SCRIPT_ROOT_PTR),
-        **inputs.tx_args().tx_script().as_ref().unwrap().hash()
+        **inputs.tx_args().tx_script().as_ref().unwrap().hash(),
+        "The transaction script root should be stored at the TX_SCRIPT_ROOT_PTR"
     );
 }
 
 fn block_data_memory_assertions(process: &Process<MockHost>, inputs: &PreparedTransaction) {
-    // The block hash should be stored at the BLK_HASH_PTR
     assert_eq!(
         read_root_mem_value(process, BLK_HASH_PTR),
-        inputs.block_header().hash().as_elements()
```

Comment:
Moved the comments to the assert because if it gives a better error message if the test fails.